pkgs/by-name/de/decent-sampler/package.nix

```diff
@@ -10,22 +12,40 @@
 
 let
   pname = "decent-sampler";
-  version = "1.9.4";
+  version = "1.10.0";
 
-  decent-sampler = stdenv.mkDerivation {
+  decent-sampler = stdenv.mkDerivation rec {
     inherit pname version;
 
     src = fetchzip {
       # dropbox link: https://www.dropbox.com/sh/dwyry6xpy5uut07/AABBJ84bjTTSQWzXGG5TOQpfa\
-
       url = "https://archive.org/download/decent-sampler-linux-static-download-mirror/Decent_Sampler-${version}-Linux-Static-x86_64.tar.gz";
-      hash = "sha256-lTp/mukCwLNyeTcBT68eqa7aD0o11Bylbd93A5VCILU=";
+      hash = "sha256-KYCf/F2/ziuXDHim4FPZQBARiSywvQDJBzKbHua+3SM=";
+    };
+
+    icon = fetchurl {
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/decent-sampler.png?h=decent-sampler-bin";
```

Comment:
This file doesn't display on my machine; I just see a black square. Also, is this not fetchable from elsewhere?

Reply:
Thanks have incorporated all your suggestions. Also changed `categories` to `[ "Audio" "AudioVideo" ]` in the desktopitem as it was giving a warning otherwise. 

Not sure what to do about the icon. The icon is basically a small black "d s" which is admittedly very difficult to see due to both size & colour. I don't think it's a huge deal coming from the AUR (there seem to be a lot of AUR links in nixpkgs), and the internet archive dropbox-mirror was set up by the AUR maintainer. So this whole package is somewhat dependent on AUR. Having said that, aesthetically I agree, the visibility of the icon is terrible. I found one or two other much better light coloured "d s" icons, but either from the apple app store, or facebook, or some third party website. I could possibly grab an icon from one of those places, but then would have to host it somewhere. Let me know what you think.

Many thanks for the feedback. Much appreciated!

Reply:
> This file doesn't display on my machine; I just see a black square. Also, is this not fetchable from elsewhere (rather than from the AUR package, which is maintained by an unknown 3rd party and might disappear)?

I've replaced the icon with another one. Should be more visible and hosted on internet archive.